packages/cli/lib/validators/marketplaceValidators/module/ModuleDependencyValidator.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path');
+
+const { logger } = require('@hubspot/cli-lib/logger');
+const { HUBSPOT_FOLDER } = require('@hubspot/cli-lib/lib/constants');
+const { fetchModuleDependencies } = require('@hubspot/cli-lib/api/marketplace');
+const { isRelativePath } = require('@hubspot/cli-lib/path');
+
+const RelativeValidator = require('../RelativeValidator');
+const { VALIDATOR_KEYS } = require('../../constants');
+
+class ModuleDependencyValidator extends RelativeValidator {
+  constructor(options) {
+    super(options);
+
+    this.errors = {
+      FAILED_TO_FETCH_DEPS: {
+        key: 'failedDepFetch',
+        getCopy: ({ filePath }) =>
+          `Internal Error. Failed to fetch dependencies for ${filePath}. Please try again`,
+      },
+      EXTERNAL_DEPENDENCY: {
+        key: 'externalDependency',
+        getCopy: ({ filePath, referencedFilePath }) =>
+          `External dependency. ${filePath} references a file (${referencedFilePath}) that is outside of the module's immediate folder.`,
+      },
+      ABSOLUTE_DEPENDENCY_PATH: {
+        key: 'absoluteDependencyPath',
+        getCopy: ({ filePath, referencedFilePath }) =>
+          `Relative path required. ${filePath} references a file (${referencedFilePath}) using an absolute path`,
+      },
+    };
+  }
+
+  failedToFetchDependencies(err, relativePath, validationErrors) {
+    logger.debug(
+      `Failed to fetch dependencies for ${relativePath}: `,
+      err.error
+    );
+
+    validationErrors.push(
+      this.getError(this.errors.FAILED_TO_FETCH_DEPS, relativePath)
+    );
+  }
+
+  async getAllDependenciesByPath(relativePath, accountId, validationErrors) {
+    let deps = [];
+    const file_deps = await fetchModuleDependencies(
+      accountId,
+      relativePath
+    ).catch(err => {
+      console.log(err);
```

Comment:
Is this log intentionally here? If so, the typical pattern in the CLI is to use `logger.debug` for logging out errors like this. Alternatively, you could define a lyaml string for this scenario with some sort of explanation.

Reply:
Good catch, accidentally left those in from testing 👀 

Don't think we need to `logger.debug`, if these errors trip then they'll show up as validation errors.